src/Randomizer.App/StringColorConverter.cs

```diff
@@ -1,29 +1,27 @@
 ﻿using System;
+using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 using System.Globalization;
 using System.Windows.Data;
 using System.Windows.Media;
 
 namespace Randomizer.App
 {
-    [ValueConversion(typeof(Color), typeof(string))]
+    [ValueConversion(typeof(List<byte>), typeof(string))]
```

Comment:
I assume this is because Color is a WPF-specific class?

List<byte> is a little confusing, `byte[]` would be more appropriate. A Color class of our own would be best, but that's probably a little bit too much for how little we actually use colors.

Reply:
Yeah, I tried adding in the framework to the data project after the fact since it wasn't added originally since it was just a library project, and... the App project really didn't seem to like that. So this was sort of the path of least resistance.

Switched to an array, though. The change corrupted my settings config, but I think that shouldn't happen for other folks.